src/pages/Components/ComponentsInfo/LinksInfo.jsx

```diff
@@ -131,6 +131,26 @@ const linkListHtml = `
 </a>
 `;
 
+const tableLink = `
+<TableLink title="Tabell 1" href="" hrefText="Gjennomsnittspris i fritt salg på eid tomt i 100-meterbeltet. 2015-2108." />
```

Comment:
I noticed that the 'Link list' section above had all of the code lines down (despite being the same) to illustrate a point. Should the React and HTML code for 'Tabell 2' also be included then, or does the code for 'Tabell 1' suffice?